packages/webapp/src/containers/AddTask/TaskAssignment/index.js

```diff
@@ -61,8 +65,30 @@ export default function TaskManagement({ history, match }) {
     setWageData(wageData.concat(wage_data));
   }, []);
 
+  const getHarvestTasksData = (taskData) => {
+    let harvestTasks = [];
+    for (let harvest_task of taskData.harvest_tasks) {
+      let id = harvest_task.id.split('.');
+      let location = id[0];
+      let managementPlan = id[1];
+      let h = {};
+      h.location_id = location;
+      h.management_plan_id = Number(managementPlan);
+      h.harvest_everything = harvest_task.harvest_everything;
+      h.quantity = harvest_task.quantity === undefined ? 0 : harvest_task.quantity;
```

Comment:
could probably stick with just checking `harvest_task.quantity` rather than using `=== undefined`. same applies to the line below here

Reply:
Not sure what you mean. Like just `harvest_task.quantity ? .... : ... `  ?

Reply:
Yea. Would that work?

Reply:
No, that would check it's null only.

Reply:
alright then, can just resolve this